anomalo/table_resource.go

```diff
@@ -96,8 +96,8 @@ func (r *tableResource) Schema(_ context.Context, _ resource.SchemaRequest, resp
 				Validators: []validator.String{
 					// These are example validators from terraform-plugin-framework-validators
 					stringvalidator.RegexMatches(
-						regexp.MustCompile(`^(daily|data_freshness_gated)$`),
-						"Check cadence type must be null, 'daily', or 'data_freshness_gated'",
+						regexp.MustCompile(`^(daily|data_freshness_gated|observability_daily_at_x)$`),
```

Comment:
Food for thought - it might be a good idea for me to get rid of this validation for cases like this where the Anomalo API adds supported options before someone can add them here. 

Thoughts?

Reply:
Meh, you're probably in the clear for a while. If we hit it again, lets circle back

Reply:
Cool. It's definitely a nicer experience when using it to hit validation errors in the `plan` instead of `apply`, so this is preferable if it'll be stable